README.md

```diff
@@ -121,7 +121,6 @@ This repository contains a curated list of awesome open source libraries that wi
 
 ## Model and Data Versioning
 * [Aim](https://github.com/aimhubio/aim) ![](https://img.shields.io/github/stars/aimhubio/aim?style=social) - A super-easy way to record, search and compare AI experiments.
-* [Apache Marvin](https://github.com/apache/incubator-marvin) ![](https://img.shields.io/github/stars/apache/incubator-marvin.svg?style=social) is a platform for model deployment and versioning that hides all complexity under the hood: data scientists just need to set up the server and write their code in an extended jupyter notebook.
```

Comment:
Why removing marvin? Seems may be slipping into not as widely maintained but it's not archived so given still backed by apache foundation would assume still active